tests/test_ensembles.py

```
@@ -80,3 +83,295 @@ def check_any_shuffles_the_same(arr: np.ndarray) -> bool:
+    with patch(
+        "viral.ensemble_reactivation.degrees_to_cm",
+        side_effect=lambda x, _: np.array(x),
+    ):
+        with patch("viral.utils.get_wheel_circumference_from_rig", return_value=1):
+            with patch(
+                "viral.ensemble_reactivation.compute_speed_grosmark",
+                side_effect=lambda position: np.ones_like(position, dtype=float) * 10,
+            ):
+                with patch(
+                    "viral.ensemble_reactivation.gaussian_filter1d",
+                    side_effect=lambda input, sigma, axis: input,
```

Ruff (0.14.8)

Unused lambda argument: `sigma`

(ARG005)